### PR TITLE
[TASK] Poppover for Inline-Code

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Index.rst [hunk 1]
```diff
@@ -45,3 +45,6 @@ to extend the tool.
     Developer/Index
     Migration/Index
     KnownProblems/Index
+
```
Comment (garvinhicking) [inline Documentation/Index.rst:48]: This probably wasn't planned to commit? 😅

Comment (linawolf) [reply Documentation/Index.rst:48]: woopsie

**Comment by fe-breuer** [comment]
(We're checking this out now)

**Comment by linawolf** [comment]
I think it looks good like that!

**Comment by linawolf** [comment]
I think I would prefer version 2

**Comment by garvinhicking** [comment]
Please check our implementation - the warnings that happen I'm not sure why?

**Comment by linawolf** [comment]
The warnings seem to be resolved by the latest commit

**Comment by linawolf** [comment]
@garvinhicking I cannot vote on this as I initially made the PR

**Review by garvinhicking — APPROVED** [review]

**Comment by linawolf** [comment]
@garvinhicking There seem to be conflicst

**Comment by garvinhicking** [comment]
@linawolf Don't see a conflict here, let me know if there is anything I can brush up!

**Pull request merged**